forc/GFS/README.md

```diff
@@ -0,0 +1,94 @@
+## Description
+
+* This package generates DATM input data for the forcing fluxes at the sea surface from the NCEP operational GFSv16 outputs.
+
+## Inputs of conversion in conv_gfs2datm_long_beta4.f
+
+* INPUT Sources : from the output of GFSv16,
+
+I. ./gdas.${dat}/${hh}/atmos/gdas.t${hh}z.sfluxgrbf000.grib2 :: Input data are from the GFSv16 outs of initial analysis fields, 
+  except following missing variables in the initial file.
+
+II. ./gdas.${datb}/${hhb}/atmos/gdas.t${hhb}z.sfluxgrb${hprep}.grib2 :: 6 hour forecast fields at the analysis time
+      ==> PRATE_surface, UFLX_surface, VFLX_surface
```

Comment:
do we need to list these here as they are repeated in the table.

Reply:
No it does not. Remove it